scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/KryoCoder.scala

```diff
@@ -27,3 +29,15 @@ object KryoCoder {
   implicit def castType[T](kryoCoder: KryoCoder): AtomicCoder[T] =
     kryoCoder.asInstanceOf[AtomicCoder[T]]
 }
+
+case class OrderedSerializationCoder[T](ordSer: OrderedSerialization[T])(implicit kryoCoder: KryoCoder) extends AtomicCoder[T] {
+  override def encode(value: T, outStream: OutputStream): Unit = ordSer.write(outStream, value)
+  override def decode(inStream: InputStream): T = ordSer.read(inStream).get
+}
+object OrderedSerializationCoder {
+  def apply[T](ord: Ordering[T])(implicit kryoCoder: KryoCoder): AtomicCoder[T] =
```

Comment:
nit: not a `KryoCoder`, and better to call it fallback. Also no need to have it as implicit parameter here